src/components/Card/Card.js

```diff
@@ -1,10 +1,17 @@
-import React from 'react';
+import React, { useState } from 'react';
 import "./Card.css";
+import Modal from '../Modal';
 
-export const Card = ({pokemon}) => {
+export const Card = ({pokemon }) => {
+  const [isOpenFlag, setIsOpenFlag] = useState(false);
+  function handleCard (openFlag){
```

Comment:
いいですね！
今回の課題の内容ではありませんがこちらはさらに以下のような形でカスタムフック化できそうですのでお時間ある際にお試しください👀

```suggestion
const useModal = () => {
  const [isOpen, setIsOpen] = useState(false);
  
  const open = () => setIsOpen(true);
  const close = () => setIsOpen(false);
  
  return { isOpen, open, close };
}
```